docs/source/modes.rst

```diff
@@ -43,6 +43,15 @@ Simple Simulation
 
 The simple simulation case is the default mode. Its usage is explained in :ref:`Getting Started`. Every chain of modes starts with a simple simulation, even if it is not explicitly listed in the modes. The simulation takes the scenario as is. No parameters will be adjusted, optimized or changed in any way. The charging type for each vehicle is read from the rotation information from the trips.csv if this data is included. If the data is not included *preferred_charging_type* from the config file is used, as long as the provided vehicles data provides the preferred_charging_type for the specified vehicle type.
 
+Load Pickle
+-----------
+
+::
+
+    mode = ["load_pickle"]
+
+Instead of simulating given scenario, read in pickle file instead. Needs the `load_pickle` option to point to a pickle file. If the `load_pickle` option is set, `load_pickle` must be the first mode. However, this mode may come in later in the mode chain again, reloading schedule and scenario from file. Options in config file that are stored in schedule are ignored.
```

Comment:
Instead of simulating ("a" or "the") given scenario.
außerdem
warum "instead"? Der mode "report" ist ja auch nicht "instead of simulating... a report is generated."
Mein Vorschlag wäre mehr bezug zum Laden des szenarios nehmen.
"Instead of generating a scenario based on file for trip, vehicle and station data, a previously simulated scenario is loaded. This scenario already contains results. Modes can be chained after load_pickle in the usual fashion. Multiple "load_pickle" modes per chain are allowed....

Reply:
# However, cost parameters are supposed to be mutable after loading from pickle
sollte auch erwähnt werden

Reply:
Rewritten